proto/message_api/v3/mls.proto

```diff
@@ -0,0 +1,161 @@
+// Message API
+syntax = "proto3";
+package xmtp.message_api.v3;
```

Comment:
I just skipped right over V2 and went straight to V3 for naming consistency with other V3 protos